includes/init_includes/init_languages.php

```diff
@@ -20,6 +20,7 @@
   } else {
     if (LANGUAGE_DEFAULT_SELECTOR=='Browser') {
       $lng->get_browser_language();
+      if (!zen_not_null($lng->language['id'])) $lng->set_language(DEFAULT_LANGUAGE);

```

Comment:
Debating whether to incorporate this fallback into get_browser_language() directly.